web/style/modal.css

```diff
@@ -0,0 +1,41 @@
+.modal-wrapper {
```

Comment:
nit: I find it helpful to roughly sort the CSS declaration lines for a block into related groups, e.g. font, size, borders, background, etc. If you think this would be helpful for you as well feel free to apply it, otherwise no worries, just a thought!

Reply:
Yep that makes sense - will help avoid the repetition as well like the above mistake that I made.

Sorted